src/SonarScanner.MSBuild.PreProcessor/SonarWebService.cs

```diff
@@ -439,21 +436,14 @@ private static string GetProjectIdentifier(string projectKey, string projectBran
             return projectId;
         }
 
-        private string GetUrl(string format, params string[] args)
-        {
-            var queryString = EscapeQuery(format, args);
-            if (!queryString.StartsWith("/", StringComparison.OrdinalIgnoreCase))
-            {
-                queryString = string.Concat('/', queryString);
-            }
+        private Uri GetUri(string query, params string[] args) =>
+            new(serverUrl, Escape(query, args));
 
-            return this.serverUrl + queryString;
-        }
+        private static string Escape(string format, params string[] args) =>
+            string.Format(CultureInfo.InvariantCulture, format, args.Select(Escape).ToArray());
```

Comment:
I'm not sure about the purpose of `CultureInfo.InvariantCulture` here.
None of the URLs contains culture variant strings.
If any do, it should depend on explicit formatting anyway (like `{0:yyyy-MM-dd}`) instead of hoping for the best framework behavior with `{0:d}`.

Reply:
I'm also not sure about it so I don't want to change it now